test/factories/author.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :author do
+    sequence(:name) {|n| "Author #{n}" }
+    description { "#{name} description\n" }
+
+    site
+  end
+
+  factory :author_with_photo, parent: :author do
+    photo { Rack::Test::UploadedFile.new(Rails.root + "test/fixtures/images/face.png", "image/png") }
```

Comment:
Carrierwave official wiki: https://github.com/carrierwaveuploader/carrierwave/wiki/How-to:-Use-test-factories